tasks/agent.py

```diff
@@ -173,6 +173,10 @@ def refresh_assets(ctx, build_tags, development=True, puppy=False):
         # copy the dd-agent placeholder to the bin folder
         bin_ddagent = os.path.join(BIN_PATH, "dd-agent")
         shutil.move(os.path.join(dist_folder, "dd-agent"), bin_ddagent)
+
+    # Network tracer not supported on windows
+    if sys.platform.startswith('linux'):
+      shutil.copy("./cmd/agent/dist/network-tracer.yaml", os.path.join(dist_folder, "network-tracer.yaml"))
```

Comment:
You should add `network-tracer.yaml` to the `.gitignore` section "# go-generated files"

Reply:
Ah good catch. Updated.